### PR TITLE
fix: add title to show what the Control icons do

### PR DESCRIPTION
**Pull request by di-uni**
## Related Issue
#1384 

<br/>

## Implementation
Previously, it wasn't able to get the information before clicking the Control icon.
So, add title attribute to show what the Control icons do.

<br/>

## Screenshot

https://user-images.githubusercontent.com/50884017/188849124-2a0cb21a-6d80-4bc3-9a7f-f1dce176def8.mov

It showed nothing when I screenshot, so I upload the recording.

<br/>

## Discussion Required
Refer to language converter(ko.json) 
if it exists, I used it, but if it does not exist, write it in English

### CODE REVIEW

File: src/components/backend-ai-storage-list.ts [hunk 1]
```diff
@@ -1244,6 +1244,7 @@ export default class BackendAiStorageList extends BackendAIPage {
           <mwc-icon-button
             class="fg green controls-running"
             icon="info"
+            title="info"
```
Comment (agatha197) [inline src/components/backend-ai-storage-list.ts]: How about using i18n strings like others?
Here's a simple way to use the i18n string.
1. Modify [ko.json or en.json](https://github.com/lablup/backend.ai-webui/tree/main/resources/i18n) files.
2. Run `make i18n`
3. Translate other words if you can.

More details: https://github.com/lablup/backend.ai-webui#extracting-i18n-resources

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-storage-list.ts [hunk 2]
```diff
@@ -1252,6 +1253,7 @@ export default class BackendAiStorageList extends BackendAIPage {
               <mwc-icon-button
                 class="fg blue controls-running"
                 icon="folder_open"
+                title="open folder"
```
Comment (agatha197) [inline src/components/backend-ai-storage-list.ts]: Same as above.

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-storage-list.ts (hunk 1 above)
Comment (di-uni) [reply src/components/backend-ai-storage-list.ts]: Ok I will try it!

Comment (di-uni) [reply src/components/backend-ai-storage-list.ts]: I resolve it. Can you review again?

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM!

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**